### PR TITLE
[CTOR-609] refresh Protocol TCP doc page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 1]
```diff
@@ -5,48 +5,51 @@ title: TCP Protocol
...
-Le connecteur apporte les modèles de service suivants (classés par modèle d'hôte):
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
 
 <Tabs groupId="sync">
-<TabItem value="App-Protocol-Tcp" label="App-Protocol-Tcp">
+<TabItem value="App-Protocol-Tcp-custom" label="App-Protocol-Tcp-custom">
 
 | Alias | Modèle de service | Description |
 |:------|:------------------|:------------|
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md:24]:
```suggestion
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 2]
```diff
@@ -5,48 +5,51 @@ title: TCP Protocol
...
-| N/A   | N/A               | N/A         |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **App-Protocol-Tcp-custom** est utilisé.
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md:26]:
```suggestion
Aucun modèle de service n'est lié à ce modèle d'hôte.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 3]
```diff
@@ -5,48 +5,51 @@ title: TCP Protocol
...
 
 </TabItem>
-<TabItem value="App-Protocol-Tcp-Only" label="App-Protocol-Tcp-Only">
+<TabItem value="App-Protocol-Tcp-Only-custom" label="App-Protocol-Tcp-Only-custom">
 
 | Alias | Modèle de service | Description |
 |:------|:------------------|:------------|
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md:32]:
```suggestion
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 4]
```diff
@@ -5,48 +5,51 @@ title: TCP Protocol
...
-| N/A   | N/A               | N/A         |
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **App-Protocol-Tcp-Only-custom** est utilisé.
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md:34]:
```suggestion
Aucun modèle de service n'est lié à ce modèle d'hôte.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 5]
```diff
@@ -124,242 +127,256 @@ que vous ne souhaitez pas découvrir des éléments pour la première fois, alor
...
-|             | TIMEOUT      | Connection timeout in seconds (Default: 3)                                      | 5       |
-|             | PORT         | Port used                                                                       |         |
-|             | WARNING      |                                                                                 |         |
-|             | CRITICAL     |                                                                                 |         |
-|             | EXTRAOPTIONS | Any extra option you may want to add to the command line (eg. a --verbose flag) |         |
+| Macro        | Description                                                                                        | Valeur par défaut | Obligatoire |
+|:-------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| TIMEOUT      | Connection timeout in seconds (default: 3)                                                         | 5                 |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md:210]: description said 3 and default value 5

Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md:210]:
```suggestion
| TIMEOUT      | Connection timeout in seconds                                                         | 5                 |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 6]
```diff
@@ -124,242 +127,256 @@ que vous ne souhaitez pas découvrir des éléments pour la première fois, alor
...
-|             | WARNINGRTMAX  |                                                                                 |         |
-|             | CRITICALRTMAX |                                                                                 |         |
-|             | WARNINGRTMIN  |                                                                                 |         |
-|             | CRITICALRTMIN |                                                                                 |         |
-|             | EXTRAOPTIONS  | Any extra option you may want to add to the command line (eg. a --verbose flag) |         |
+| Macro         | Description                                                                                        | Valeur par défaut | Obligatoire |
+|:--------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| PACKETS       | Number of packets to send (default: 5)                                                             | 5                 |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md:221]:
```suggestion
| PACKETS       | Number of packets to send                                                            | 5                 |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 7]
```diff
@@ -124,242 +127,256 @@ que vous ne souhaitez pas découvrir des éléments pour la première fois, alor
...
+| TIMEOUT       | Set timeout in seconds (default: 5)                                                                | 5                 |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md:222]:
```suggestion
| TIMEOUT       | Set timeout in seconds                                                                | 5                 |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 8]
```diff
@@ -124,242 +127,256 @@ que vous ne souhaitez pas découvrir des éléments pour la première fois, alor
...
+| PORT          | Port used                                                                                          |                   | X           |
+| WARNINGPL     | Packets lost threshold warning in %                                                                |                   |             |
+| CRITICALPL    | Packets lost threshold critical in %                                                               |                   |             |
+| WARNINGRTA    | Response time threshold warning in milliseconds                                                    |                   |             |
+| CRITICALRTA   | Response time threshold critical in milliseconds                                                   |                   |             |
+| WARNINGRTMAX  | Warning maximum response time threshold                                                                                                   |                   |             |
+| CRITICALRTMAX | Critical maximum response time threshold
+                                                                                                   |                   |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md:230]:
```suggestion
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 9]
```diff
@@ -5,48 +5,50 @@ title: TCP Protocol
...
 
-The connector brings the following service templates (sorted by host template):
+The connector brings the following service templates (sorted by the host template they are attached to):
 
 <Tabs groupId="sync">
-<TabItem value="App-Protocol-Tcp" label="App-Protocol-Tcp">
+<TabItem value="App-Protocol-Tcp-custom" label="App-Protocol-Tcp-custom">
 
 | Service Alias | Service Template  | Service Description |
 |:--------------|:------------------|:--------------------|
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-protocol-tcp.md:23]:
```suggestion
No service template is linked to this host template.
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 10]
```diff
@@ -5,48 +5,50 @@ title: TCP Protocol
...
-| N/A           | N/A               | N/A                 |
+
+> The services listed above are created automatically when the **App-Protocol-Tcp-custom** host template is used.
 
 </TabItem>
-<TabItem value="App-Protocol-Tcp-Only" label="App-Protocol-Tcp-Only">
+<TabItem value="App-Protocol-Tcp-Only-custom" label="App-Protocol-Tcp-Only-custom">
 
 | Service Alias | Service Template  | Service Description |
 |:--------------|:------------------|:--------------------|
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-protocol-tcp.md:31]:
```suggestion
No service template is linked to this host template.
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 11]
```diff
@@ -126,227 +128,244 @@ You still have to manually install the plugin on the poller(s) when:
...
-|             | TIMEOUT      | Connection timeout in seconds (Default: 3)                                      | 5       |
-|             | PORT         | Port used                                                                       |         |
-|             | WARNING      |                                                                                 |         |
-|             | CRITICAL     |                                                                                 |         |
-|             | EXTRAOPTIONS | Any extra option you may want to add to the command line (eg. a --verbose flag) |         |
+| Macro        | Description                                                                                        | Default value     | Mandatory   |
+|:-------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| TIMEOUT      | Connection timeout in seconds (default: 3)                                                         | 5                 |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-protocol-tcp.md:211]:
```suggestion
| TIMEOUT      | Connection timeout in seconds                                                         | 5                 |             |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 12]
```diff
@@ -126,227 +128,244 @@ You still have to manually install the plugin on the poller(s) when:
...
-|             | WARNINGRTMAX  |                                                                                 |         |
-|             | CRITICALRTMAX |                                                                                 |         |
-|             | WARNINGRTMIN  |                                                                                 |         |
-|             | CRITICALRTMIN |                                                                                 |         |
-|             | EXTRAOPTIONS  | Any extra option you may want to add to the command line (eg. a --verbose flag) |         |
+| Macro         | Description                                                                                        | Default value     | Mandatory   |
+|:--------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| PACKETS       | Number of packets to send (default: 5)                                                             | 5                 |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-protocol-tcp.md:222]:
```suggestion
| PACKETS       | Number of packets to send                                                            | 5                 |             |
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 13]
```diff
@@ -126,227 +128,244 @@ You still have to manually install the plugin on the poller(s) when:
...
+| TIMEOUT       | Set timeout in seconds (default: 5)                                                                | 5                 |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/applications-protocol-tcp.md:223]:
```suggestion
| TIMEOUT       | Set timeout in seconds                                                                 | 5                 |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 14]
```diff
@@ -124,242 +123,255 @@ que vous ne souhaitez pas découvrir des éléments pour la première fois, alor
...
-<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
 
 ```bash
 dnf install centreon-plugin-Applications-Protocol-Tcp
 ```
 
 </TabItem>
-<TabItem value="CentOS 7" label="CentOS 7">
+<TabItem value="Debian 11" label="Debian 11">
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-tcp.md:140]:
```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-tcp.md [hunk 15]
```diff
@@ -126,227 +126,244 @@ You still have to manually install the plugin on the poller(s) when:
...
-<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
 
 ```bash
 dnf install centreon-plugin-Applications-Protocol-Tcp
 ```
 
 </TabItem>
-<TabItem value="CentOS 7" label="CentOS 7">
+<TabItem value="Debian 11" label="Debian 11">
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-tcp.md:143]:
```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**